source/simulator/parameters.cc

```diff
@@ -167,7 +167,7 @@ namespace aspect
                        "Units: Years or seconds, depending on the ``Use years "
                        "in output instead of seconds'' parameter.");
 
-    prm.declare_entry ("Maximum relative increase in time step", boost::lexical_cast<std::string>(std::numeric_limits<int>::max()),
+    prm.declare_entry ("Maximum relative increase in time step", "86.0",
                        Patterns::Double (0.),
                        "Set a percentage with which the time step is limited to increase. Generally the "
                        "time step based on the CFL number should be sufficient, but for complicated models "
```

Comment:
Would you mind adding a sentence to the end of the description of the parameter to explain where that 86% choice comes from? It will probably not be obvious to anyone why that is a useful choice, and we developers will likely forget about it as well.

More specifically, I think it would be useful if you could add the reasoning you provided in the comments in the pull request to the description of the parameter. People will read the manual where this information is replicated, but will not go back to the pull request where the functionality was added.